ci/run.sh

```diff
@@ -8,8 +8,17 @@ if [ "$1" = "aarch64-apple-darwin" ] ; then
 	export CARGO_TARGET_AARCH64_APPLE_DARWIN_RUNNER=echo
 fi
 
-cargo test --manifest-path testcrate/Cargo.toml --target $1 -vv
-cargo test --manifest-path testcrate/Cargo.toml --target $1 -vv --release
+# Use cargo package to ensure we don't rely on any files excluded by Cargo.toml
+cargo package --allow-dirty
+
+version=$(cargo run output-version)
+
+testcrate_dir="$(pwd)/testcrate"
+
+cd "target/package/openssl-src-$version"
```

Comment:
To avoid adding a new binary just to print the version, could this do `cd target/package/openssl-src-*`?

Reply:
Hmm. Even in the naïve case where we're starting with a clean `target/` directory we have two matching entries (the directory and the .crate archive), so this will fail. We can delete *.crate to make it work, however. We should ensure `target/package` is emptied first though, since it relies on the assumption that we're starting with an empty directory.

Reply:
I like `target/ci` so I've just done that